### PR TITLE
feat: option to center top menu

### PR DESCRIPTION
**Pull request by davifantasia**
Closes https://github.com/commercetools/commercetools-docs-kit/issues/538

### CODE REVIEW

File: packages/gatsby-theme-docs/src/components/top-menu.js [hunk 1]
```diff
@@ -169,7 +177,7 @@ const TopMenu = () => {
           event.stopPropagation();
         }}
       >
-        <ContentGrid>
```
Comment (emmenko) [inline packages/gatsby-theme-docs/src/components/top-menu.js:172]: This grid component should stay, as all the children are positioned accordingly to the grid. If you change the container to a block element, the children will be break the layout most likely.

File: packages/gatsby-theme-docs/src/components/top-menu.js [hunk 2]
```diff
@@ -169,7 +177,7 @@ const TopMenu = () => {
...
+        <div css={props.centered ? centeredContainerStyle : contentGridStyle}>
```
Comment (davifantasia) [inline packages/gatsby-theme-docs/src/components/top-menu.js:180]: @emmenko the grid still remains but through css prop now.

Comment (emmenko) [reply packages/gatsby-theme-docs/src/components/top-menu.js:180]: But the centered container style is not a grid 🧐

Comment (davifantasia) [reply packages/gatsby-theme-docs/src/components/top-menu.js:180]: I'm not sure what you mean, but currently, the children's layout is not affected from what I see - I get exactly what I want by making it a block element, and children are properly centered.

File: packages/gatsby-theme-docs/src/components/top-menu.js (hunk 1 above)
Comment (nkuehn) [reply packages/gatsby-theme-docs/src/components/top-menu.js:172]: @emmenko I just inspected the built site: The content has its own grid in the "Columns" component.  

This "ContentGrid" is an extra wrapper to position it the same way the main page content is positioned.

**Pull request merged**